### ISSUE TITLE
Add option to not keep screen on

### ISSUE THREAD

**Issue by bonbonboi** [stefan-niedermann/nextcloud-notes#1531]
You forcibly added a feature that we don't want because of that please add an option to not to keep screen on.

**Comment by mjanssens** [comment]
Different people have different needs.
This change was a feature request, see https://github.com/stefan-niedermann/nextcloud-notes/issues/1461
So it wasn't forcibly added. And it's not so nice too say that "we" don't want this, because you should than clarify who else is writing this with you ;-)
But I assume that other than that your are a happy user of this app, just like me.

I can understand that you were a bit annoyed, but best way is to ask the developer nicely for another feature request or change/adjustment of the implementation.

What guided me to this issue:
Often, I leave my notes open for a long while. So after this change https://github.com/stefan-niedermann/nextcloud-notes/commit/d721cb128d837f8fb15e3d55565439c09fa44ef5 I noticed that my battery had drained a lot because my screen stayed on and also the screen didn't lock automatically. So for that, it would indeed be nice to have a choice in the settings to toggle keeping screen on or not. So I would rephrase the title of this issue as: Add option to toggle keep screen on or not.

**Comment by stefan-niedermann** [comment]
While I personally see no use case of disabling the display while viewing or editing a note, the impact on the code is quite low and the maintainability will not be effected that much.

I therefore added an option to `3.5.1` which will be published in the next few weeks.
